src/firebase.js

```diff
@@ -0,0 +1,50 @@
+import { initializeApp } from "firebase/app";
+import * as mess from "firebase/messaging";
+// import { getMessaging, getToken, onMessage } from "firebase/messaging";
+
+var firebaseConfig = {
+
+    apiKey: "AIzaSyBAW9eosfF-oo5xPXXeoBp810YLyFlYZJw",
+  
+    authDomain: "testing-uci-web-channel.firebaseapp.com",
+  
+    projectId: "testing-uci-web-channel",
+  
+    storageBucket: "testing-uci-web-channel.appspot.com",
+  
+    messagingSenderId: "584834206650",
+  
+    appId: "1:584834206650:web:b1a8a5a03dd231406e2b83",
+  
+    measurementId: "G-WSYRRR972G"
+  
+  };
+
+  
+export const getToken = (setTokenFound) => {
+    return mess.getToken(messaging, {vapidKey: 'BKuweez_2-hT2OYFRwUZ6CUuH3B3wrY0y1EOWlnbYtML0Iy7okQQZiiFZwzIYoin5TqdglT-18k-ki5cY_aUyjc'}).then((currentToken) => {
+      if (currentToken) {
+        console.log('current token for client: ', currentToken);
```

Comment:
Remove console.log.

Reply:
Add an eslint rule that prohibits console.log

Reply:
![image](https://user-images.githubusercontent.com/76895787/170890621-a13bc2b1-7274-4f66-8387-b91fda861299.png)
Can you confirm if this is the right rule to do so

Reply:
and would commenting the `console.log` be suffice or should I totally remove it?

Reply:
This is fine. We can modify this later if needed. If we can keep `warn` and `error` as warning, that would be great. `log` would be at error level.

Reply:
Btw I have currently commented out this rule because there are some files other than mine which include `console.log` and I didn't know if I should remove their `console.log` or not